client/question/services/FeedbackGeneratorService.js

```diff
@@ -53,6 +69,23 @@ tie.factory('FeedbackGeneratorService', [
     var consecutiveSameRuntimeErrorCounter = 0;
 
     /**
+     * Object to hold index of correctness feedback given (values), per
+     * test suite (keys). Used when avoiding presenting the same feedback text
```

Comment:
A test suite is an object, so it can't be a key. Do you mean test suite ID, test case key, or something else?

What does "index of correctness feedback given" mean?

Reply:
Fixed